### DISCUSSION TITLE
Not able to signup in documenso in local setup

### DISCUSSION THREAD

**Discussion by sumitbishti** [documenso/documenso#831] [General]
I am not able to signup in documenso. Getting this error. I have also setup NEXTAUTH_SECRET but same error.
 
[next-auth][error][JWT_SESSION_ERROR] 
@documenso/web:dev: https://next-auth.js.org/errors#jwt_session_error decryption operation failed {
@documenso/web:dev:   message: 'decryption operation failed',
@documenso/web:dev:   stack: 'JWEDecryptionFailed: decryption operation failed\n' +
@documenso/web:dev:     '    at gcmDecrypt (webpack-internal:///(rsc)/../../node_modules/jose/dist/node/cjs/runtime/decrypt.js:68:15)\n' +
@documenso/web:dev:     '    at decrypt (webpack-internal:///(rsc)/../../node_modules/jose/dist/node/cjs/runtime/decrypt.js:91:20)\n' +
@documenso/web:dev:     '    at flattenedDecrypt (webpack-internal:///(rsc)/../../node_modules/jose/dist/node/cjs/jwe/flattened/decrypt.js:137:52)\n' +
@documenso/web:dev:     '    at async compactDecrypt (webpack-internal:///(rsc)/../../node_modules/jose/dist/node/cjs/jwe/compact/decrypt.js:20:23)\n' +
@documenso/web:dev:     '    at async jwtDecrypt (webpack-internal:///(rsc)/../../node_modules/jose/dist/node/cjs/jwt/decrypt.js:10:23)\n' +
@documenso/web:dev:     '    at async Object.decode (webpack-internal:///(rsc)/../../node_modules/next-auth/jwt/index.js:44:25)\n' +
@documenso/web:dev:     '    at async Object.session (webpack-internal:///(rsc)/../../node_modules/next-auth/core/routes/session.js:25:34)\n' +      
@documenso/web:dev:     '    at async AuthHandler (webpack-internal:///(rsc)/../../node_modules/next-auth/core/index.js:161:37)\n' +
@documenso/web:dev:     '    at async getServerSession (webpack-internal:///(rsc)/../../node_modules/next-auth/next/index.js:126:21)\n' +
@documenso/web:dev:     '    at async eval (webpack-internal:///(rsc)/../../packages/lib/next-auth/get-server-component-session.ts:23:21)\n' +       
@documenso/web:dev:     '    at async NotFound (webpack-internal:///(rsc)/./src/app/not-found.tsx:18:25)',
@documenso/web:dev:   name: 'JWEDecryptionFailed'
@documenso/web:dev: }

<img width="715" alt="Screenshot 2024-01-14 161314" src="https://github.com/documenso/documenso/assets/75713174/412775e2-771e-4027-859f-d6380684266b">

**Comment by Mythie** [comment]
Hey there, can I confirm that all the database migrations have ran?

You can check by running `prisma:migrate-dev` or `primsa:migrate-deploy` depending on your environment, for local dev the `prisma:migrate-dev` command will suffice.

**Comment by sumitbishti** [comment]
yes db migrations ran perfectly fine

Getting this error at console:
 ⨯ Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './build/src/interfaces' is not defined by "exports" in C:\Users\sumit\Desktop\documenso\node_modules\resend\package.json
@documenso/web:dev:     at new NodeError (node:internal/errors:405:5)
@documenso/web:dev:     at exportsNotFound (node:internal/modules/esm/resolve:259:10)
@documenso/web:dev:     at packageExportsResolve (node:internal/modules/esm/resolve:589:9)
@documenso/web:dev:     at resolveExports (node:internal/modules/cjs/loader:569:36)
@documenso/web:dev:     at Module._findPath (node:internal/modules/cjs/loader:643:31)
@documenso/web:dev:     at Module._resolveFilename (node:internal/modules/cjs/loader:1056:27)
@documenso/web:dev:     at C:\Users\sumit\Desktop\documenso\apps\web\node_modules\next\dist\server\require-hook.js:54:36
@documenso/web:dev:     at Module._load (node:internal/modules/cjs/loader:923:27)
@documenso/web:dev:     at Module.require (node:internal/modules/cjs/loader:1137:19)
@documenso/web:dev:     at mod.require (C:\Users\sumit\Desktop\documenso\apps\web\node_modules\next\dist\server\require-hook.js:64:28)
@documenso/web:dev:     at require (node:internal/modules/helpers:121:18)
@documenso/web:dev:     at Object.<anonymous> (C:\Users\sumit\Desktop\documenso\node_modules\@documenso\nodemailer-resend\dist\main.js:4:18)
@documenso/web:dev:     at Module._compile (node:internal/modules/cjs/loader:1255:14)
@documenso/web:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
@documenso/web:dev:     at Module.load (node:internal/modules/cjs/loader:1113:32) {
@documenso/web:dev:   code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
@documenso/web:dev: }

**Comment by cangsalak** [comment]
1. Open docker desktop as well.
2. Load data from github
3. Open terminal, type `npm i`
4. When finished, type `npm run d`
5. When finished, type `npm run prisma:migrate-dev`
6. When finished, type `npm run dev`
It's finished.

**Comment by sumitbishti** [comment]
I have done every step apart from step 2. How to do it?
